benchmarks/src/main/scala/scalaz/zio/IODeepAttemptBenchmark.scala

```diff
@@ -28,14 +28,12 @@ class IODeepAttemptBenchmark {
 
   @Benchmark
   def futureDeepAttempt(): BigInt = {
-    import scala.util.Success
     import scala.concurrent.Future
     import scala.concurrent.duration.Duration.Inf
 
     def descend(n: Int): Future[BigInt] =
       if (n == depth) Future.failed(new Exception("Oh noes!"))
-      else if (n == halfway) descend(n + 1).transform(_.transform(Success(_), _ => Success(50)))
-      else descend(n + 1).map(_ + n)
+      else if (n == halfway) descend(n + 1).recover { case _ => 50 } else descend(n + 1).map(_ + n)
```

Comment:
Was this change necessary or just cosmetic? Did `Future` API changed? Just asking, I'm ok if it's cosmetic, looks much nicer this way.
Have you run the benchmarks to compare to previous results and see if it still makes sense?

Reply:
2.11's Future doesn't have `def transform[S](f: Try[T] => Try[S])`

I've run/compared the jmh benchmarks briefly, they look similar enough.

Reply:
I guess, to be fair with all the competitors, if `transform` was chosen b/c of performance reasons we should split that benchmark into 2.11 and 2.12 implementations. It can be a separate PR, if that makes sense at all, coz `Future` is way way behind in the numbers.

Reply:
Maybe chosen to avoid partial functions rather than for performance.